CHANGELOG.md

```diff
@@ -1,3 +1,103 @@
+### citus v9.5.0 (November 10, 2020) ###
+
+* Adds support for PostgreSQL 13
+
+* Removes the task-tracker executor
+
+* Introduces citus local tables
+
+* Introduces `undistribute_table` UDF to convert tables back to postgres tables
+
+* Adds support for `EXPLAIN ANALYZE EXECUTE` and `EXPLAIN EXECUTE`
+
+* Adds support for `EXPLAIN(ANALYZE, WAL)` for PG13
+
+* Adds support for `WITH TIES` option in SELECT and INSERT SELECT queries
+
+* Avoids taking multi-shard locks on workers
+
+* Enforces `citus.max_shared_pool_size` config in COPY queries
+
+* Enables custom aggregates with multiple parameters to be executed on workers
+
+* Enforces `citus.max_intermediate_result_size` in local execution
+
+* Improves cost estimation of INSERT SELECT plans
+
+* Introduces delegation of procedures that read from reference tables
+
+* Prevents pull-push execution for simple pushdownable subqueries
+
+* Sorts the output of `EXPLAIN ANALYZE` by execution duration.
+
+* Disallows `ON TRUE` outer joins with ref & dist tables when ref table is outer
+
+* Disallows field indirection in INSERT/UPDATE queries
+
+* Disallows volatile functions in UPDATE subqueries
+
+* Fixes ALTER TABLE IF EXISTS SET SCHEMA bug with non-existing tables
+
+* Fixes CREATE INDEX CONCURRENTLY crash with local execution
+
+* Fixes CREATE INDEX CONCURRENTLY with no index name crash with postgres tables
+
+* Fixes `citus_finish_pg_upgrade` to drop all backup tables
+
+* Fixes a bug that cause failures when `RECURSIVE VIEW` joined reference table
+
+* Fixes DROP SEQUENCE failures when metadata syncing is enabled
+
+* Fixes a bug that caused CREATE TABLE with CHECK constraint to fail
+
+* Fixes a bug that caused VACUUM to deadlock in PG13
```

Comment:
Seems irrelevant since we introduced PG13 support here

Reply:
That's true. But if I understood correctly, @marcocitus thought it might worth to mention as this is also changing the behavior for pg11&12 as well. Removing seems to make more sense, what do you think @marcocitus @JelteF  ?

Reply:
If it's reworded to also apply to pg11 & pg12 I think it makes sense to keep.

Reply:
then rewording it to `Fixes a bug that could cause VACUUM to deadlock` at 9ae6e4d34ddb8008f7356671cf0c5dd17cc171a5